recipes/DebuggingLog/README.md

```diff
@@ -0,0 +1,9 @@
+# DebuggingLog
+
+This recipe shows how to do print-debugging using the `Debug` module's `spy` and `traceM` functions.
+
+## Expected Behavior:
+
+Does console-printing-based debugging in various contexts to show the pros/cons of each approach and how reliable/unpredictable they can be.
```

Comment:
You might also want to include that these functions include a compiler warning, which helps ensure this code gets removed in say the code review process rather than make it to production.

It's similar to why some folks define a custom `todo` function like this:

```purs
todo
  :: forall sym a
   . IsSymbol sym
  => Warn (Beside (Text "TODO: ") (Text sym))
  => SProxy sym
  -> a
todo sym =
  unsafeThrowException $ error $ "TODO: " <> reflectSymbol sym
```

which allows you to make a TODO comment that also includes a compiler warning so you don't forget it.

Reply:
Good point. I've updated the readme to include a note about this, so that it shows up in the recipe ToC, too.